app/api/__init__.py

```diff
@@ -41,7 +42,7 @@
 
 # tickets
 api.route(AllTicketList, 'all_ticket_list', '/tickets', '/events/<int:id>/tickets')
-api.route(TicketDetail, 'ticket_detail', '/tickets/<int:id>', '/events/<int:event_id>/tickets')
```

Comment:
This end point makes no sense. Since Event to ticket is one to many relationship.